beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -51,53 +51,67 @@ func (s *Server) GetAggregateAttestation(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	if err := s.AttestationsPool.AggregateUnaggregatedAttestations(ctx); err != nil {
```

Comment:
found out this was just wrong before in the gRPC implementation of GetAggregateAttestation, instead we should be looking at the business logic of `SubmitAggregateSelectionProof` in v1alpha1